code/modules/library/lib_machines.dm

```diff
@@ -322,7 +322,10 @@ GLOBAL_LIST(cachedbooks) // List of our cached book datums
 	if (prob(50))
 		new /obj/item/melee/cultblade/dagger(get_turf(src))
 		to_chat(user, "<span class='warning'>Your sanity barely endures the seconds spent in the vault's browsing window. The only thing to remind you of this when you stop browsing is a sinister dagger sitting on the desk. You don't even remember where it came from...</span>")
-
+	else
+		new /obj/item/clockwork/clockwork_slab(get_turf(src))
+		to_chat(user, "<span class='warning'>Your sanity barely endures the seconds spent in the vault's browsing window. The only thing to remind you of this when you stop browsing is a strange metal tablet sitting on the desk. You don't even remember where it came from...</span>")
+		
```

Comment:
Why not also add heretic book as well with two prob(33)s and an else?

Reply:
It'd be far better to have a switch statement at that point.